src/main/kotlin/it/pagopa/helpdeskcommands/utils/RedirectKeysConfiguration.kt

```diff
@@ -0,0 +1,95 @@
+package it.pagopa.helpdeskcommands.utils
+
+import arrow.core.Either
+import arrow.core.right
+import it.pagopa.helpdeskcommands.exceptions.RedirectConfigurationException
+import it.pagopa.helpdeskcommands.exceptions.RedirectConfigurationType
+import java.net.URI
+import java.util.*
+import java.util.function.Predicate
+import java.util.stream.Collectors
+
+/**
+ * This class manages the PSP redirect URL map. It has been converted to Kotlin from the
+ * ecommerce-commons Java class
+ */
+class RedirectKeysConfiguration(
+    pspUrlMapping: Map<String, String>,
+    paymentTypeCodeList: Set<String>
+) {
+    private val redirectBeApiCallUriMap: Map<String, URI>
+
+    init {
+        val redirectUriMap: MutableMap<String, URI> = HashMap()
+        // URI.create throws IllegalArgumentException that will prevent module load for
```

Comment:
you can set local configuration like https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/31da939b251e93e1b75792e82a325f26c52b73da/.env.example#L75

Reply:
and update application.properties

Reply:
Added local configuration with commit [905948](https://github.com/pagopa/pagopa-ecommerce-helpdesk-commands-service/pull/36/commits/905948edd04bd6c9c29a041f000e519c2560e73b)